lib/snippet_helpers.rb

```diff
@@ -61,8 +67,11 @@ def language
     end
   end
 
-  def code_for(snippet_name)
-    partial 'code', locals: { snippets: snippets_for(snippet_name) }
+  def code_for(snippet_name, variable_name = "refills-variables")
+    partial 'code', locals: {
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.